src/modules/design/property-panel/indextabs/general/sections/data-objects/data-output-association/data-output-association.scss

```
@@ -0,0 +1,28 @@
+.data-output-association-enlarge-link {
+  display: block;
+  color: #007bff !important;
+}
+
+.data-output-association-enlarge-link:hover {
+  text-decoration: underline !important;
+}
+
+.data-output-association-enlarge-text:hover {
+  text-decoration: underline;
+}
+
+.data-output-association {
+  min-height: 55px;
+  max-height: 400px;
+  min-width: 470px;
+  max-width: 100%;
+  resize: both;
+}
+
+.data-source-input {
+  height: 100px !important;
+}
+
+.information-text {
```

```suggestion
.information-text {
```

Ich würde hier einen weniger allgemeinen Namen verwenden.
Solch allgemeine CSS Klassennamen können immer gefährlich sein, weil dadurch Inhalte von anderen Komponenten auf einmal anders aussehen können nur weil auf einmal dieser Code geladen wurde.